cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -461,7 +461,7 @@ impl<Bridges: ExporterFor, Router: SendXcm, UniversalLocation: Get<InteriorLocat
 	fn validate(
 		dest: &mut Option<Location>,
 		msg: &mut Option<Xcm<()>>,
-		_source: Option<&Location>,
+		source: Option<&Location>,
```

Comment:
Add the source location(optional) to `SendXcm` trait so that we can `DescendOrigin` to the souce location(i.e. user) later.